logos/src/logos/main.py

```
@@ -2894,6 +2897,101 @@ async def delete_user(user_id: int, request: Request):
+@app.get("/teams", tags=["teams"])
+async def list_teams(request: Request):
+    logos_key = require_app_admin_or_above(request)
+    with DBManager() as db:
+        caller = db.get_user_by_logos_key(logos_key)
+        if caller["role"] == "logos_admin":
+            return db.list_teams()
+        return db.list_teams(owner_user_id=caller["id"])
+
+
+@app.post("/teams", tags=["teams"])
+async def create_team(body: CreateTeamRequest, request: Request):
```

Flake8 (7.3.0)

[error] 'CreateTeamRequest' may be undefined, or defined from star imports: logos.dbutils.dbrequest

(F405)